install_pim/manual/system_requirements/system_requirements.rst.inc

```diff
@@ -82,9 +92,9 @@ Besides these modules, the following configuration is the minimal configuration
 
 **Database server**
 
-+-----------------+---------------------+
-| MySQL (SQL)     | >= 5.7.22 and < 8.0 |
-+-----------------+---------------------+
++-----------------+----------+
+| MySQL (SQL)     | = 8.0.16 |
```

Comment:
for now, as long as the bug https://bugs.mysql.com/bug.php?id=96823 exists
next, I'll put something like `>= 8.0.18 and < 8.1`

Reply:
`>= 8.0.16 and < 8.1` then

Reply:
no because the bug is on 8.0.17 AFAIK